lib/multer-extended.module.ts

```diff
@@ -32,6 +32,7 @@ export class MulterExtendedModule {
   public static registerAsync(options: MulterExtendedS3AsyncOptions): DynamicModule {
     return {
       module: MulterExtendedModule,
+      imports: options.imports || [],
```

Comment:
@gimyboya 
Actually, `|| []` is not necessary.
`imports: options.imports` is fine. :)